command/server.go

```diff
@@ -245,6 +247,23 @@ func (c *ServerCommand) Flags() *FlagSets {
 			"production.",
 	})
 
+	f.BoolVar(&BoolVar{
+		Name:   "dev-tls",
+		Target: &c.flagDevTLS,
+		Usage: "Enable TLS development mode. In this mode, Vault runs in-memory and " +
+			"starts unsealed, with a generated TLS CA, certificate and key. " +
+			"As the name implies, do not run \"dev-tls\" mode in " +
+			"production.",
+	})
+
+	f.StringVar(&StringVar{
+		Name:       "dev-tls-cert-dir",
+		Target:     &c.flagDevTLSCertDir,
+		Default:    "",
+		Completion: complete.PredictDirs("*"),
+		Hidden:     true,
```

Comment:
Is there a reason to keep this hidden? Perhaps we should expose it and document it along with `-dev-tls`?

Reply:
Bad copy-paste. I removed the flag.